src/python_testing/TC_AVSUMTestBase.py

```diff
@@ -0,0 +1,95 @@
+#
```

Comment:
General Q's - is there a way to validate that the camera actually pans or tilts or zooms as expected?
And is there a way to validate that it PTZ's in the right direction?

Also, you're probably not there yet, but how does the digital PTZ factor in here? Can you double down?